xds/internal/xdsclient/xdsresource/unmarshal_cds_test.go

```diff
@@ -194,6 +194,23 @@ func (s) TestValidateCluster_Failure(t *testing.T) {
 			wantUpdate: emptyUpdate,
 			wantErr:    true,
 		},
+		{
+			name: "aggregate-v3-empty-clusters",
```

Comment:
Nit: Please skip v3 in the name. We have already dropped support for v2, and we should cleanup any mentions of v2/v3 in consts/symbol-names as an ongoing effort.

Reply:
Done.